android/app/src/main/java/com/expensify/chat/customairshipextender/CustomNotificationListener.java

```diff
@@ -26,6 +26,10 @@ public void onNotificationPosted(@NonNull @NotNull NotificationInfo notification
 
     @Override
     public boolean onNotificationOpened(@NonNull @NotNull NotificationInfo notificationInfo) {
+        // The notification is also dismissed when it's tapped so handle that as well
+        PushMessage message = notificationInfo.getMessage();
+        provider.onDismissNotification(message);
```

Comment:
Ah nice. But do you mind removing the change from this PR? We already have an issue for this [here](https://github.com/Expensify/App/issues/22972)

Reply:
Feel free to take over the issue if you want. Up to you

Reply:
Yeah I already tested the issue so I can takeover